highfidelity/hifi
# MS17463: only show InteractiveWindow on top of interface window

fixes https://highfidelity.manuscript.com/f/cases/17463/InteractiveWindows-stay-on-top-when-interface-not-focussed

## QA Test 
1. be in desktop mode
2. run script test  `scripts/developer/test/interactiveWindowTest.js`
3. a native window should pop up with the text in title bar "Native Desktop Window", and in center of the window you should see "OverlayWindowTestString"
4. switch to VR mode
5. the native window should disappear from the desktop and re-appear on the Virtual desktop with title "Virtual Desktop Window".
6. switch back to Desktop mode. Verify that you see the same result as in step 3.
7. Platform specific step:
     - On MS Windows the window should be always in front of interface and yet other windows should be able to move in front of the window. (Try moving your web browser over the other window, the interactive window should not interfere with your web browsing.)
      - On other platforms the interactive window will always overlap over any window.
7. stop the script. -> the window should disappear.
8. smoke test other already existing UI elements/windows.
